Install instructions don't match reality

### Akaunting version

latest

### PHP version

latest

### Operating system

Debian 12

### Steps to reproduce

Follow install instruction on GitHub

### Expected result

The install for composer and node (using nvm) are redirected to their respective websites.  They give inaccurate instruction and result in installation of tools incompatible with this project.

Dependencies are missing: git, php, working C/C++ toolchain (build-essential) + others

### Actual result

Fails to install

### Additional comments

_No response_

### Relevant log output

```shell

```

Got the same issue. 
For PHP: Using PHP 8.3.23 and got this error when running `composer install`
```#18 /Users/longkhoa/fun/akaunting/vendor/symfony/console/Command/Command.php(326): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#19 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Console/Command.php(180): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#20 /Users/longkhoa/fun/akaunting/vendor/symfony/console/Application.php(1096): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /Users/longkhoa/fun/akaunting/vendor/symfony/console/Application.php(324): Symfony\Component\Console\Application->doRunCommand(Object(Barryvdh\LaravelIdeHelper\Console\ModelsCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 /Users/longkhoa/fun/akaunting/vendor/symfony/console/Application.php(175): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(201): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 /Users/longkhoa/fun/akaunting/artisan(22): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#25 {main}
Exception: SQLSTATE[HY000] [2002] Connection refused
Could not analyze class App\Models\Common\Notification.

Trace:
#0 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(65): PDO->__construct('mysql:host=127....', 'forge', Object(SensitiveParameterValue), Array)
#1 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(83): Illuminate\Database\Connectors\Connector->createPdoConnection('mysql:host=127....', 'forge', '', Array)
#2 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php(48): Illuminate\Database\Connectors\Connector->tryAgainIfCausedByLostConnection(Object(PDOException), 'mysql:host=127....', 'forge', '', Array)
#3 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connectors/MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=127....', Array, Array)
#4 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connectors/ConnectionFactory.php(184): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#5 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#6 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connection.php(1339): call_user_func(Object(Closure))
#7 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connection.php(1288): Illuminate\Database\Connection->getPdo()
#8 /Users/longkhoa/fun/akaunting/vendor/laravel/framework/src/Illuminate/Database/Connection.php(1272): Illuminate\Database\Connection->getDoctrineConnection()
#9 /Users/longkhoa/fun/akaunting/vendor/barryvdh/laravel-ide-helper/src/Console/ModelsCommand.php(517): Illuminate\Database\Connection->getDoctrineSchemaManager()
#10 /Users/longkhoa/fun/akaunting/vendor/barryvdh/laravel-ide-helper/src/Console/ModelsCommand.php(292): Barryvdh\LaravelIdeHelper\Console\ModelsCommand->getPropertiesFromTable(Object(App\Models\Common\Notification))
#11 /Users/longkhoa/fun/akaunting/vendor/barryvdh/laravel-ide-helper/src/Console/ModelsCommand.ph```
For JS: I tried node version from 16.x to 20.x and could not install the package
```npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/no-asm/include/openssl/crmf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/crypto/buildinf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_digests.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_dsa.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_ecx.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_rsa.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_wrap.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_sm2.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/providers/common/include/prov/der_ec.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/progs.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/crypto/bn_conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/crypto/dso_conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/opensslv.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/ess.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/ocsp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/srp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/x509_vfy.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/cms.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/x509v3.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/asn1.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/safestack.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/lhash.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/ssl.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/err.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/crypto.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/cmp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/ct.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/x509.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/ui.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/fipskey.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/pkcs7.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/bio.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/asn1t.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/configuration.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/pkcs12.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/aix64-gcc-as/asm_avx2/include/openssl/crmf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/crypto/buildinf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_digests.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_dsa.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_ecx.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_rsa.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_wrap.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_sm2.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/providers/common/include/prov/der_ec.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/progs.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/crypto/bn_conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/crypto/dso_conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/opensslv.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/ess.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/ocsp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/srp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/x509_vfy.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/cms.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/x509v3.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/asn1.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/safestack.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/lhash.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/ssl.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/err.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/crypto.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/cmp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/ct.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/x509.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/ui.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/fipskey.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/pkcs7.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/bio.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/asn1t.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/configuration.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/pkcs12.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/asm/include/openssl/crmf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/crypto/buildinf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_digests.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_dsa.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_ecx.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_rsa.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_wrap.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_sm2.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/providers/common/include/prov/der_ec.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/progs.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/crypto/bn_conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/crypto/dso_conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/opensslv.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/ess.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/ocsp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/srp.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/x509_vfy.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/cms.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/conf.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/x509v3.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/asn1.h
npm ERR! gyp verb extracted file from tarball node-v18.19.0/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/safestack.h```

--- a/README.md
+++ b/README.md
@@ -27,7 +27,20 @@ Akaunting uses [Laravel](http://laravel.com), the best existing PHP framework, a
 
 ## Installation
 
-* Install [Composer](https://getcomposer.org/download) and [Npm](https://nodejs.org/en/download)
+Before installing Akaunting, make sure your environment has the required dependencies installed:
+
+* PHP 8.1 or higher with the required PHP extensions
+* Composer
+* Node.js and npm
+* Git
+* A supported database server, such as MariaDB, MySQL, PostgreSQL, or SQLite
+* A web server, such as Apache, Nginx, or IIS
+* Build tools required by some npm packages, such as `build-essential` on Debian/Ubuntu systems
+
+For the full list of PHP extensions and server requirements, see the [on-premise requirements](https://akaunting.com/hc/docs/on-premise/requirements/).
+
+Then install Akaunting:
+
 * Clone the repository: `git clone https://github.com/akaunting/akaunting.git`
 * Install dependencies: `composer install ; npm install ; npm run dev`
 * Install Akaunting:

